### ISSUE TITLE
IINA use alot of cpu after play finished.

### ISSUE THREAD

**Issue by xpader** [iina/iina#2416]
**System and IINA version:**
- macOS 10.14.4
- IINA 1.0.3

**Expected behavior:**
Eveytime I use IINA to play a video, when video is play finished, and IINA is idle, The IINA process using high cpu about 80%-100%, more than playing a video.

Everytime that happend, very stranger.

**Comment by wasp898** [comment]
Well, when the video finishes IINA tries to play whatever other video finds up next in the queue or in the same folder. Maybe that other video causes the issue for you?

**Comment by xpader** [comment]
> Well, when the video finishes IINA tries to play whatever other video finds up next in the queue or in the same folder. Maybe that other video causes the issue for you?

I don't think this is the reason, because I have closed IINA window (click left top close button) after video play finished.

**Comment by xpader** [comment]
There's same issue in #2123
